chart/templates/proxy_injector.yaml

```diff
@@ -4,6 +4,24 @@
 ### Proxy Injector
 ###
 ---
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
as a (recently-created) convention, we typically keep cluster-level resources in the `linkerd install config`. everything in `linkerd install control-plane` is scoped to the namespace. what's the motivation for including it in the `control-plane` stage?